### PR TITLE
[v10] Desktop Discovery guide

### PR DESCRIPTION
**Pull request by alexfornuto**
Manual backport of #17602, with adjustments to the config file examples to use v2.

### CODE REVIEW

File: docs/pages/desktop-access/getting-started.mdx [hunk 1]
```diff
@@ -1,468 +1,134 @@
...
-New > Certificate Template to Issue
-```
-Select `RemoteDesktopAccess` and click `OK`.
-
-### Update GPO to use a new certificate template
+Copy and paste the provided command into a Windows PowerShell window to download
+and run the 'configure Active Directory' script:
 
-In the group policy editor for `Teleport Access Policy`, select:
+![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:64]: I think it's probably fine, but technically this screenshot shows a config v3 which is not supported on Teleport 10. Up to you whether you want to go through the effort to get a v10 screenshot.

**Review by zmb3 — APPROVED** [review]
Looks good. Not sure if it's worth putting up a version notice, but this is only supported in newer versions of v10.2.8 and later.

File: docs/pages/desktop-access/getting-started.mdx (hunk 1 above)
Comment (alexfornuto) [reply docs/pages/desktop-access/getting-started.mdx:64]: I think it's fine. Folks can't copy/paste from an image after all.

File: docs/pages/desktop-access/getting-started.mdx [hunk 2]
```diff
@@ -1,468 +1,134 @@
...
+description: Automatically add Teleport Desktop Access to your infrastructure
 ---
 
-# Getting Started
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.2.6"
```
Comment (alexfornuto) [inline docs/pages/desktop-access/getting-started.mdx:11]:
```suggestion
  min="10.2.8"
```

File: docs/pages/desktop-access/getting-started.mdx [hunk 3]
```diff
@@ -1,468 +1,134 @@
...
+>
 
-In this guide we will connect an Active Directory domain to Teleport using
-Desktop Access and log into a Windows desktop from that domain.
+The features documented on this page are available in Teleport `10.2.6` and higher.
```
Comment (alexfornuto) [inline docs/pages/desktop-access/getting-started.mdx:14]:
```suggestion
The features documented on this page are available in Teleport `10.2.8` and higher.
```

File: docs/pages/desktop-access/manual-setup.mdx [hunk 4]
```diff
@@ -0,0 +1,631 @@
...
+description: Manually Connect Teleport to Active Directory and log into a Windows Desktop
+videoBanner: YvMqgcq0MTQ
+---
+
+In this guide we will connect an Active Directory domain to Teleport using
+Desktop Access and log into a Windows desktop from that domain.
+
+<Admonition type="tip" title="Use The Wizard">
+
+Starting with Teleport 10.2.6, you can install Active Directory and configure Teleport Desktop Access through the web UI. See [Getting Started](getting-started.mdx) for more information.
```
Comment (alexfornuto) [inline docs/pages/desktop-access/manual-setup.mdx:12]:
```suggestion
Starting with Teleport 10.2.8, you can install Active Directory and configure
Teleport Desktop Access through the web UI. See [Getting Started](getting-started.mdx)
for more information.
```

File: docs/pages/desktop-access/getting-started.mdx (hunk 3 above)
Comment (zmb3) [reply docs/pages/desktop-access/getting-started.mdx]: Whoops, looks like I was wrong. 10.2.6 is correct. Sorry for the runaround!

**Comment by alexfornuto** [comment]
> Looks good. Not sure if it's worth putting up a version notice, but this is only supported in newer versions of v10.2.8 and later.

I updated the notes from 102.6 to 10.2.8. Will have to do that in master/v11 as well.

**Review by ptgott — APPROVED** [review]

**Pull request merged**